classes/items/itembase.cpp

```diff
@@ -18,11 +18,7 @@ void Item::generate()
 
 
 //Constructors / Destructors
-Item::Item(std::string name,
-            unsigned int type,
-            unsigned int tier,
-            unsigned int value,
-            unsigned int level);
```

Comment:
This wasn't hte issue lines. The lines I referenced before were the ones in the comment on that line.